ui/src/pages/author-guide.tsx

```diff
@@ -73,6 +74,54 @@ const AuthorGuide: NextPage = (): React.ReactElement => (
                                 </Components.Link>
                             </>
                         </StandardText>
+                        <StandardText>
+                            <>
+                                You might want to have a look at some example publications of the different types before
+                                writing your own.{' '}
+                                <Components.Link
+                                    href="https://www.octopus.ac/faq#pub_type_octopus"
+                                    openNew={true}
+                                    className="rounded text-teal-600 outline-0 transition-colors duration-500 focus:ring-2 focus:ring-yellow-400 dark:text-teal-300"
+                                >
+                                    Here
+                                </Components.Link>{' '}
+                                is a handy guide to the eight types. It is most likely that your first publication will
+                                be of a Research Problem.{' '}
+                                <Components.Link
+                                    href="https://www.octopus.ac/publications/gv8n-ap77"
+                                    openNew={true}
+                                    className="rounded text-teal-600 outline-0 transition-colors duration-500 focus:ring-2 focus:ring-yellow-400 dark:text-teal-300"
+                                >
+                                    Here
+                                </Components.Link>{' '}
+                                is an example of what a Research Problem publication might look like. You can see other
+                                publication types linked to that one.
+                            </>
+                        </StandardText>
+                        <StandardText>
+                            <>
+                                When you&apos;re ready to publish, you will need to have to hand:
+                                <ul className="ml-8 list-disc">
```

Comment:
This list should probably be wrapped in its own <StandardText> element like the other one starting on line 129 for example, so that it gets the same spacing from the text above it.

Reply:
Good spot - thanks!